docs/tasks/access-kubernetes-api/extend-api-third-party-resource.md

```diff
@@ -14,13 +14,24 @@ redirect_from:
 
 ## What is ThirdPartyResource?
 
-**WARNING: ThirdPartyResources are not yet considered stable, and the API and/or storage could change before GA.
-Development and outstanding issues are tracked at [https://github.com/kubernetes/features/issues/95](https://github.com/kubernetes/features/issues/95).**
+**WARNING: ThirdPartyResources will be removed in 1.8 without access to existing data!  See the section on migration to CustomResourceDefinitions.**
```

Comment:
According to the [deprecation policy](https://kubernetes.io/docs/reference/deprecation-policy/), it seems like we can't remove TPR except by removing all of `extensions/v1beta1`. One could argue that maybe the extensions group is an exception because it was filled with unrelated things before the policy of domain-specific groups, but we should make sure there is consensus on that.

In terms of docs style, we should also [avoid making promises about the future](https://kubernetes.io/docs/home/contribute/style-guide/#avoid-statements-about-the-future) whenever possible. 
For now, perhaps we should merely say, "TPR is deprecated as of 1.7 and will be removed in a future release. Please migrate to CustomResourceDefinition.".

We should also make "migrate to CRD" a link to the appropriate section.